doc/whats_new.rst

```diff
@@ -44,6 +44,10 @@ Enhancements
 Bug fixes
 .........
 
+   - :class:`sklearn.feature_selection.SelectFdr` now correctly works
```

Comment:
how about"Fixed a bug where :class:feature_selection.SelectFdr did not accurately implement Benjamini-Hochberg procedure"